runtimes/cmake/Modules/FindLibcCommonUtils.cmake

```diff
@@ -0,0 +1,13 @@
+# //===--------------------------------------------------------------------===//
+# //
+# // Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# // See https://llvm.org/LICENSE.txt for details.
+# // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# //
+# //===--------------------------------------------------------------------===//
```

Comment:
Nitpick: license looks weird. For CMake or Python we generally use just `#`-comments, no `//` needed.